### PR TITLE
Update Neuro_Annotations.json

### PR DESCRIPTION
**Pull request by amapeters**

### CODE REVIEW

File: Neuro_Annotations.json [hunk 1]
```diff
@@ -176,5 +179,17 @@
...
+    ],
+    "pH": [
+   
+    ],
+    "hemisphere": [
+        "Left",
+        "Right"
+        
+    ],
+    "BrodmannArea": [
```
Comment (kdaily) [inline Neuro_Annotations.json]: malformed JSON - missing closing square brace

File: Neuro_Annotations.json [hunk 2]
```diff
@@ -176,5 +179,17 @@
...
-    ]
+    ],
+    "PMI": [
+         
+    ],
+    "pH": [
+   
+    ],
+    "hemisphere": [
+        "Left",
```
Comment (kdaily) [inline Neuro_Annotations.json]: use lowercase

File: Neuro_Annotations.json [hunk 3]
```diff
@@ -89,6 +89,9 @@
     "sampleName": [
         
     ],
+    "individual": [
```
Comment (kdaily) [inline Neuro_Annotations.json]: did we decide to stay with `individual`, or use `individualId` in combination with `individualIdSource`?

Comment (amapeters) [inline Neuro_Annotations.json]: we decided to use individual, but individualID makes more sense I thin in terms of having and individualSource. Let's change it to individualID

**Pull request merged**